common/dev/ina233.c

```diff
@@ -48,47 +66,77 @@ uint8_t ina233_read(uint8_t sensor_num, int *reading)
 
 	ret = i2c_master_read(&msg, retry);
 	if (ret != 0) {
-		printf("[%s] i2c read fail  ret: %d\n", __func__, ret);
+		LOG_ERR("read func: i2c read fail  ret: %d\n", ret);
```

Comment:
Remove log message newline.
Remove "read func: " (is included by logging service if enabled.)